### ISSUE TITLE
Add support for JSON-LD format

### ISSUE THREAD

**Issue by ffont** [AudioCommons/ac-mediator#17]
We want to provide valid linked data format (json-ld) that can load Audio Commons responses into a triplestore. See initial experiments in branch https://github.com/AudioCommons/ac-mediator/tree/jsonld_output

We can use this issue to discuss how to implement that and make experiments.

**Comment by ffont** [comment]
Related commits so far:
37d9dc6c2c6e7f7c1a564a600aad30d3983a0108
008fe4ef677a0736e478a69169dfbe4570c9c72e
263d19963443fad55bb144d611b57f00e31f887c

**Comment by ffont** [comment]
See experimental implementations in gist: https://gist.github.com/ffont/d5e2a70e7ce7fc535eb717375a529d56

**Comment by ffont** [comment]
As discussed via Skype with @miguel76 and George, we will follow the next steps:

1. Design the JSON-LD output that we want to have based on a number of typical use cases
2. Write example scripts that generate such output (decoupled from the mediator)
3. Decide on a particular output/implementation method and integrate it with the mediator

In regards to step 1, I prepared this [list of example requests/responses](https://github.com/AudioCommons/ac-mediator/wiki/Example-responses-for-endpoints-(basic-JSON-output)) from the current implementation of the Audio Commons API which exemplify use cases.

All implementation is expected to be finished by January 2018 so that we can do the first public release of the mediator. I created a [first public release](https://github.com/AudioCommons/ac-mediator/milestone/1) milestone that we can use to organise the issues related to that.
